index.js

```diff
@@ -14,28 +18,61 @@ const closeSidebar = () => {
     blurScreenStyle.display = 'none'
 }
 
+const changeSize = (items, itemsCount) => {
+    let myMap = new Map()
+    myMap.set(2, 'two-items')
+        .set(3, 'three-items')
+    for (let elem of myMap) {
+        if (itemsCount % elem[0] === 0) {
+            items.forEach(item => {
+                item.classList.remove('two-items', 'three-items')
+                item.classList.add(elem[1])
+            })
+            break
+        }
+    }
+}
 
-const deleteItem = (elem) => {
-    document.getElementById(elem.parentNode.id).remove()
-
+const changeItemsFlow = () => {
     const items = document.querySelectorAll('.item')
     const itemsCount = items.length
 
-    let myMap = new Map()
-    myMap.set(0, 'two-items')
-        .set(1, 'three-items')
-
-    for (let item of items) {
-        if (!itemsCount) {
-            document.getElementById("portfolio").style.display = 'none'
-            break
-        } else if (myMap.has(itemsCount % 2)) {
-            item.classList.remove('two-items', 'three-items')
-            item.classList.add(`${myMap.get(itemsCount % 2)}`)
-        }
+    if (!itemsCount) {
+        document.getElementById("portfolio").style.display = 'none'
+        return 0
+    } else {
+        changeSize(items, itemsCount)
     }
 }
 
+const deleteItem = (elem) => {
+    document.getElementById(elem.parentNode.id).remove()
+    changeItemsFlow()
+}
+
+function showInfo(event) {
+    const par = document.createElement('p')
+    const text = event.target.parentNode.lastElementChild.textContent
```

Comment:
target could be a button, could be an icon. If it's an icon, you won't gat text to display. Please find a solution

Reply:
@silchencko thanks! Fixed it